tony-examples/tony-in-gcp/README.md

```diff
@@ -0,0 +1,524 @@
+![img](https://i.imgur.com/7vEjSTC.png=100x100)
+
+## Introduction
+
+Cloud [Dataproc](https://cloud.google.com/dataproc/) is a managed Spark and Hadoop service that lets you take advantage of open source data tools for batch

```

Comment:
Do we also need a "What is Google Cloud Platform? section? Also, what is the relationship between GCP and Dataproc?

Reply:
Added reference